holidayextras/ui-toolkit
# UXUI-398 - Create a Weather component

#### What does this PR do? (please provide any background)
- Creates a weather component, similar to those displayed on weather channels and websites like [BBC Weather](http://www.bbc.co.uk/weather/)
- Takes temperature, date, weather type and unit (°f,°c)

#### What tests does this PR have?
Tests cover...
- All weather types
- Valid and invalid dates
- Checks °F,°C are displayed

#### How can this be tested?
- First run `grunt docs` this should load up the UIToolkit locally
- If you scroll down to 'weather'  and click on the code tab you can actually edit the react code right there (though it isn't obvious - see https://github.com/holidayextras/ui-toolkit/issues/43)

#### Screenshots / Screencast
<img width="717" alt="screen shot 2015-09-09 at 16 12 37" src="https://cloud.githubusercontent.com/assets/778942/9765729/9edb3118-570d-11e5-9d56-907c14a86816.png">

#### What gif best describes how you feel about this work?
![bowie](https://media.giphy.com/media/zNjqqAjKlooNO/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
